.github/workflows/docker.yaml

```diff
@@ -0,0 +1,50 @@
+name: Docker
+on:
+  pull_request: {}
```

Comment:
Btw, I know this was originally on `push` to `main`, but it seems to be so fast due to cacheing, that I wonder if it's ok to just run this on PR? It would be helpful in testing I think.